src/mnesia_eleveldb.erl

```diff
@@ -462,10 +474,9 @@ pp_calls(I, [{M,F,A,Pos} | T]) ->
     [Pp(M,F,A,Pos)|[["\n",Spc,Pp(M1,F1,A1,P1)] || {M1,F1,A1,P1} <- T]].
 
 pp_pos([]) -> "";
-pp_pos(L) when is_integer(L) ->
-    [" (", integer_to_list(L), ")"];
```

Comment:
Out of interest, how did you or dialyzer determine that this case cannot occur? My concern is that the structure of StackTrace in pp_stack/0 might vary across OTP versions.

Reply:
Personally, I don't recall why that case was added, but presumably, it was to handle actual stack traces. The question would be if they can still occur, in the actually supported OTP versions.

Reply:
The stacktrace format is documented to be a list of two-tuples. From http://erlang.org/doc/reference_manual/errors.html#exceptions:

> Location is a (possibly empty) list of two-tuples that can indicate the location in the source code of the function. The first element is an atom describing the type of information in the second element. The following items can occur:
>
> * file The second element of the tuple is a string (list of characters) representing the filename of the source file of the function. 
> * line The second element of the tuple is the line number (an integer > 0) in the source file where the exception occurred or the function was called.

Dialyzer has a explicit clause in `erl_bif_types.erl:type/5` specifying how stacktraces look:
```erlang
type(erlang, build_stacktrace, 0, _, _Opaques) ->
  t_list(t_tuple([t_module(),
                  t_atom(),
                  t_sup([t_arity(),t_list()]),
                  t_list(t_sup([t_tuple([t_atom('file'),t_string()]),
                                t_tuple([t_atom('line'),t_pos_integer()])]))]));
```

Reply:
OTP-21 and up seem to have this doc, which is good enough for us.